package.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     {
       "name": "MoEngagePluginBase",
-      "version": "5.1.0",
+      "version": "5.1.1",
```

Comment:
If I remember correctly plugin-base requires all modules to be released, when doing any kind of release. Please verify this.

Reply:
addressed